Microsoft.Toolkit.Uwp.UI.Controls/ClassicMenu/ClassicMenuItem.cs

```diff
@@ -0,0 +1,122 @@
+        /// </summary>
+        public static readonly DependencyProperty HeaderProperty = DependencyProperty.Register(nameof(Header), typeof(string), typeof(ClassicMenuItem), new PropertyMetadata(default(string)));
+
+        /// <summary>
+        /// ClassicMenuItem Meny Style
+        /// </summary>
+        public static readonly DependencyProperty MenuStyleProperty = DependencyProperty.Register(nameof(MenuStyle), typeof(Style), typeof(ClassicMenuItem), new PropertyMetadata(default(Style)));
+
+        /// <summary>
+        /// ClassicMenuItem Menu background
+        /// </summary>
+        public static readonly DependencyProperty MenuBackgroundProperty =
+            DependencyProperty.Register(
+            nameof(MenuBackground),
+            typeof(Brush),
+            typeof(ClassicMenuItem),
+            new PropertyMetadata(default(Brush)));
+
+        private ClassicMenu ParentMenu => this.FindAscendant<ClassicMenu>();
+
+        private Button _flyoutButton;
+
+        /// <summary>
+        /// Gets or sets the title to appear in the title bar
+        /// </summary>
+        public string Header
+        {
+            get { return (string)GetValue(HeaderProperty); }
+            set { SetValue(HeaderProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the menu style for ClassicMenuItem
+        /// </summary>
+        public Style MenuStyle
+        {
+            get { return (Style)GetValue(MenuStyleProperty); }
+            set { SetValue(MenuStyleProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the MenuBackground to appear in the title bar
+        /// </summary>
+        public string MenuBackground
+        {
+            get { return (string)GetValue(HeaderProperty); }
+            set { SetValue(HeaderProperty, value); }
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ClassicMenuItem"/> class.
+        /// </summary>
+        public ClassicMenuItem()
+        {
+            DefaultStyleKey = typeof(ClassicMenuItem);
+            IsFocusEngagementEnabled = true;
+        }
+
+        /// <summary>
+        /// This method is used to show the menu for current item
+        /// </summary>
+        public void ShowMenu()
+        {
+            _flyoutButton?.Flyout?.ShowAt(_flyoutButton);
+        }
+
+        /// <inheritdoc />
+        protected override void OnApplyTemplate()
+        {
+            _flyoutButton = GetTemplateChild("FlyoutButton") as Button;
+
+            if (_flyoutButton != null && Items != null && Items.Any())
+            {
+                var menuFlyout = new MenuFlyout();
+                menuFlyout.Placement = ParentMenu.Orientation == Orientation.Horizontal
+                    ? FlyoutPlacementMode.Bottom
+                        : FlyoutPlacementMode.Right;
+                menuFlyout.MenuFlyoutPresenterStyle = MenuStyle;
+
+                foreach (var item in Items)
```

Comment:
Since this is an ItemsControl the ItemsSource could be bound to a backing collection and the items would **not** be menu flyouts. Should account for that now.